### PR TITLE
Update grant roadmap (post-Kale, pre-Callisto)

### LINKED ISSUE #128: Update Roadmap based on learning and progress made
## Summary

Once we wrap up the Kale sprint we need to update the roadmap in jupyter/accessibility to reflect the status of items and plans

### PR DESCRIPTION
**Pull request by isabela-pf**
Fixes #128 

When we do things, we need to update our public roadmap! This PR

- Removes completed items from the roadmap
- Adds the next focus for the roadmap
- Creates `done.md` as a "changelog" to collect completed items from off the roadmap. Right now, I'm summarizing by sprint name because it seemed most direct, but I'm open to other ideas.
- Adds 'done.md' to the table of contents

I based this on the work at the [Kale](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/5?closed=1), [Kale - lessons learned](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/8), and [Callisto](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/6) milestones.

### CODE REVIEW

File: docs/roadmap/done.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+:::
+
+## Completed in spring 2022 (Kale 🌱)
+
+Find the full list of grant issues at [Kale](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/5) and [Kale - lessons learned milestones.](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/8)
+
+- Maintain the [JupyterLab accessibility project board](https://github.com/orgs/Quansight-Labs/projects/5/views/1)
+- Add automated accessibility tests to [jupyter/accessibility](https://github.com/jupyter/accessibility/) for running in JupyterLab (and other configurable Jupyter interfaces) ([JupyterLab issue #9742](https://github.com/jupyterlab/jupyterlab/issues/9742) and [jupyter/accessibility #18](https://github.com/jupyter/accessibility/issues/18))
+- Begin writing custom, manual testing-inspired accessibility tests for JupyterLab
+- Create JupyterLab accessibility statment
```
Comment (gabalafou) [inline docs/roadmap/done.md:21]:
```suggestion
- Create JupyterLab accessibility statement
```

File: docs/roadmap/done.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+:::
+
+## Completed in spring 2022 (Kale 🌱)
+
+Find the full list of grant issues at [Kale](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/5) and [Kale - lessons learned milestones.](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/8)
+
+- Maintain the [JupyterLab accessibility project board](https://github.com/orgs/Quansight-Labs/projects/5/views/1)
+- Add automated accessibility tests to [jupyter/accessibility](https://github.com/jupyter/accessibility/) for running in JupyterLab (and other configurable Jupyter interfaces) ([JupyterLab issue #9742](https://github.com/jupyterlab/jupyterlab/issues/9742) and [jupyter/accessibility #18](https://github.com/jupyter/accessibility/issues/18))
+- Begin writing custom, manual testing-inspired accessibility tests for JupyterLab
+- Create JupyterLab accessibility statement
```
Comment (trallard) [inline docs/roadmap/done.md:21]: Since we had this discussion, maybe we can reword it to `Created a first iteration of...` or something of the such

File: docs/roadmap/intro.md [hunk 3]
```diff
@@ -22,27 +22,28 @@ Over the course of the two-year grant, we are aiming to transform the Jupyter ec
 
 ## Action items
 
-### :runner: In the works
```
Comment (trallard) [inline docs/roadmap/intro.md:25]: Thanks I saw it the other day and noted it mentally to fix it 🎉  but you read my mind

**Review by trallard — APPROVED** [review]
Only one comment - not blocking though

**Comment by trallard** [comment]
LGTM merging now 🚀
